patroni/postgresql.py

```diff
@@ -622,12 +629,12 @@ def restart(self):
 
     def _write_postgresql_conf(self):
         # rename the original configuration if it is necessary
-        if not os.path.exists(self._postgresql_base_conf):
+        if not config.get('custom_conf') and not os.path.exists(self._postgresql_base_conf):
```

Comment:
if 'custom_conf' not in self.config and not os.path.exists(self._postgresql_base_conf):